tests/helpers/run-command.js

```diff
@@ -43,18 +44,21 @@ module.exports = function run(/* command, args, options */) {
     options.log(`      Running: ${command} ${args.join(' ')} in: ${process.cwd()}`);
 
     let opts = {};
+    args = [`--unhandled-rejections=strict`, `${command}`].concat(args);
```

Comment:
ensure all acceptance tests that spawn node sub-processes via `run-command` also run in the same `--unhandled-rejections=strict` mode